content/chainguard/chainguard-enforce/iam-groups/rolebinding-terraform-gh/index.md

```diff
@@ -0,0 +1,393 @@
+---
+title: "Create Rolebindings for a GitHub Team Using Terraform"
+linktitle: "GitHub Team Role Binding"
+description: "Procedural tutorial outlining how to use Terraform to create Chainguard rolebindings for members of a GitHub team."
+type: "article"
+date: 2023-06-10T08:48:45+00:00
+lastmod: 2023-06-10T08:48:45+00:00
+draft: false
+tags: ["Enforce", "Product", "Procedural"]
+images: []
+weight: 050
+---
+
+There may be cases where an organization will want multiple users to have access to the same Chainguard Enforce instance. Originally, Chainguard allowed you to grant other users access to Enforce by [generating an invite link or code](/chainguard/chainguard-enforce/iam-groups/how-to-manage-iam-groups-in-chainguard-enforce/#inviting-others-to-a-group).
+
+However, this process has its limitations, as it forces you to generate invite codes for each user one at a time. It also means that you must ensure that the invite codes or links are distributed in a secure manner, or else you risk unwanted users having access to Enforce.
+
+This guide outlines one method of using Terraform to grant members of a GitHub team access to the resources managed by a Chainguard Enforce group. It also highlights a few other Terraform configurations you can use to manage rolebindings in the Chainguard platform.
```

Comment:
What about adding that while this is specific to GitHub, the same approach can be used for other systems?